deploy/shakenfist_ci/tests/test_upgrades.py

```diff
@@ -4,170 +4,18 @@
-                        'type': 'disk'
-                    }
-                ],
-                'error_message': None,
-                'memory': 1024,
-                'name': 'inst1',
-                'namespace': 'system',
-                'node': 'sf-2',
-                'power_state': 'on',
-                'ssh_key': None,
-                'state': 'created',
-                'user_data': None,
-                'uuid': 'ce2c93af-a9f8-4efe-94df-3786c3f92a7a',
-                'vdi_port': 40495,
-                'version': 2,
-                'video': {
-                    'memory': 16384,
-                    'model': 'cirrus'
-                }
-            }, self.system_client.get_instance('ce2c93af-a9f8-4efe-94df-3786c3f92a7a')
-        )
-
-        # Second instance
-        self.assertEqual(
-            {
-                'console_port': 47305,
-                'cpus': 1,
-                'disk_spec': [
-                    {
-                        'base': 'cirros',
-                        'bus': None,
-                        'size': 8,
-                        'type': 'disk'
-                    }
-                ],
-                'error_message': None,
-                'memory': 1024,
-                'name': 'inst2',
-                'namespace': 'system',
-                'node': 'sf-2',
-                'power_state': 'on',
-                'ssh_key': None,
-                'state': 'created',
-                'user_data': None,
-                'uuid': 'a9121ca7-4f72-4b94-87f4-2a1e0d54f4f4',
-                'vdi_port': 46898,
-                'version': 2,
-                'video': {
-                    'memory': 16384,
-                    'model': 'cirrus'
-                }
-            }, self.system_client.get_instance('a9121ca7-4f72-4b94-87f4-2a1e0d54f4f4')
-        )
-
-        # Third instance
-        self.assertEqual(
-            {
-                'console_port': None,
-                'cpus': 2,
-                'disk_spec': [
-                    {
-                        'base': 'ubuntu',
-                        'bus': None,
-                        'size': 20,
-                        'type': 'disk'
-                    }
-                ],
-                'error_message': None,
-                'memory': 2048,
-                'name': 'ubuntu',
-                'namespace': 'nstest',
-                'node': 'sf-2',
-                'power_state': 'initial',
-                'ssh_key': None,
-                'state': 'preflight',
-                'user_data': None,
-                'uuid': 'a13ce821-adcb-45a4-9260-1b18b5250ad3',
-                'vdi_port': None,
-                'version': 2,
-                'video': {
-                    'memory': 16384,
-                    'model': 'cirrus'
-                }
-            }, self.system_client.get_instance('a13ce821-adcb-45a4-9260-1b18b5250ad3')
-        )
+            instances['%s/%s' % (inst['namespace'], inst['name'])] = inst
```

Comment:
Same again. Check the instance has no errors.

Reply:
Sorry, done but I didn't reply.